web/src/components/project/component/list.js

```diff
@@ -1,10 +1,107 @@
 import React from 'react';
```

Comment:
About naming, please refer to `https://github.com/airbnb/javascript/tree/master/react#naming`. I suggest the initial letter for the component name be uppercase.

Reply:
I will add a CONTRIBUTING file to cover such coding conventions both for frontend and backend. And investigate tools that can help auto check.